discord/ext/commands/help.py

```diff
@@ -1088,9 +1087,60 @@ def add_indented_commands(
         for command in commands:
             name = command.name
             width = max_size - (get_width(name) - len(name))
-            entry = f'{self.indent * " "}{name:<{width}} {command.short_doc}'
+            entry = f'{self.indent * " "}{name:<{width}} - {command.short_doc}'
             self.paginator.add_line(self.shorten_text(entry))
 
+    def add_command_arguments(
+        self, command: Command[Any, ..., Any], /, *, heading: str, max_size: Optional[int] = None
```

Comment:
```suggestion
        self, command: Command[Any, ..., Any], /
```

This interface doesn't make much sense. The heading is already an attribute and the `max_size` doesn't need to be specified.

Reply:
Ah my bad, that was copied from the signature of `DefaultHelpCommand.add_indented_commands`
https://github.com/Rapptz/discord.py/blob/584c22f642cfb63b921467a901c8a47c721923b3/discord/ext/commands/help.py#L1051-L1053

Reply:
For that one, the heading used is dynamic so it makes a bit more sense. The documentation nit is still relevant there though...